packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.0.1
+
+* Adds support for retrieving navigation type with internal class.
+* Updates README with details on contributing.
```

Comment:
Let's mention the Pigeon update here, since this picked up the change in serialization format (so has a higher likelihood than usual of introducing bugs, since it's a fundamental change in Pigeon internals).